src/main/scala/io/scalac/tezos/translator/model/SendEmail.scala

```diff
@@ -24,7 +24,7 @@ object SendEmail {
       s"""
          |Please add my translation to your library:
          |Title: ${libraryDto.title}
-         |Description: ${libraryDto.description}
+         |Description: ${libraryDto.description.getOrElse("-")}
```

Comment:
Maybe a better option will be  created this "|Description: ${libraryDto.description.getOrElse("-")}" as a value dependend of libraryDto.description.

Reply:
what do you exactly suggest ?